bus-mapping/src/evm/opcodes/error_gas_uint_overflow.rs

```diff
@@ -0,0 +1,73 @@
+use super::{Opcode, OpcodeId};
+use crate::{
+    circuit_input_builder::{CircuitInputStateRef, ExecStep},
+    error::ExecError,
+    Error,
+};
+use eth_types::GethExecStep;
+
+/// Placeholder structure used to implement [`Opcode`] trait over it
+/// corresponding to the
+/// [`ExecError::GasUintOverflow`](crate::error::ExecError::GasUintOverflow).
+#[derive(Clone, Copy, Debug)]
+pub(crate) struct ErrorGasUintOverflow;
+
+impl Opcode for ErrorGasUintOverflow {
+    fn gen_associated_ops(
+        state: &mut CircuitInputStateRef,
+        geth_steps: &[GethExecStep],
+    ) -> Result<Vec<ExecStep>, Error> {
+        let geth_step = &geth_steps[0];
+        let opcode = geth_step.op;
+
+        debug_assert!([
```

Comment:
suggest to make these various unit overflow to merge with each kind of oog cases.  like unit overflow in call * merge to OOG Call circuit. unit overflow in log*, merge to OOG LOG gadget.  all of existing gadgets already handle required stack operations. we can reuse and reduce stack related operations, also helpful for future maintenance.

Reply:
looks better now.